.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
add entire .idea folder to .gitignore file and any other build generated folders like out or target